frontend/models/product-list/index.ts

```diff
@@ -412,8 +414,6 @@ function createPublicAlgoliaKey(appId: string, apiKey: string): string {
    return publicKey;
 }
 
-function createIFixitAPIClient() {
-   return new IFixitAPIClient({
-      origin: IFIXIT_ORIGIN,
-   });
+function createIFixitAPIClient(origin: string) {
```

Comment:
This function is now kinda not worth having.

Reply:
https://github.com/iFixit/react-commerce/pull/1025/commits/56cd5fcf4723cc7bafa50503d292cd568c53929d